<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeHistogramWidget</class>
 <widget class="QWidget" name="TimeHistogramWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="histoStatsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Statistics indicators</string>
     </property>
     <property name="statusTip">
      <string>Statistics indicators: mean, standard deviation and median values</string>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="histogramLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Time histogram</string>
     </property>
     <property name="statusTip">
      <string>Time histogram with a log scale on vertical axis and linear on horizontal axis</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(127, 127, 127);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="histoMaxLabel">
     <property name="statusTip">
      <string>Max value of histogram on horizontal axis</string>
     </property>
     <property name="whatsThis">
      <string>Max value of histogram on horizontal axis</string>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
